lib/passport-wsfed-saml2/wsfederation.js

```diff
@@ -59,6 +59,12 @@ WsFederation.prototype = {
       return callback(new Error('missing RequestedSecurityToken element'));
     }
 
+    // Check for more than one Assertions to prevent signature wrapping attack
+    var foundAssertions = xpath.select("//*[local-name(.)='Assertion']", token);
+    if (foundAssertions.length > 1) {
+      return callback(new Error('A RequestedSecurityToken can contain only one Assertion element.'));
```

Comment:
will this be logged as an exploitation attempt ? (versus just an error)

Reply:
Replied privately